lib/shared/index.dart

```diff
@@ -0,0 +1,11 @@
+class Index {
```

Comment:
What is the purpose of this class?

Reply:
It's a little complex, because i have to update the state of the current index (a variable) from my MovieMenu class from the CustomNavigationBar (because it's there when it changes the state of the current index). And the only way to implement that i found (maybe there is another option) is creating a new class (because i have only one instance that is passed for reference between widgets)

Reply:
Done with a callback!